client/components/Menu/Menu.tsx

```diff
@@ -3,8 +3,8 @@ import classNames from 'classnames';
 import { Classes } from '@blueprintjs/core';
 import * as RK from 'reakit/Menu';
 
-import { MenuContext, MenuConfigContext } from './menuContexts';
 import { Placement } from 'popper.js';
+import { MenuContext, MenuConfigContext } from './menuContexts';
```

Comment:
Interesting, it appears the tool organized imports a bit better (but not quite 100% our convention). Looks like an improvement though.